mirascope/core/anthropic/_utils/get_json_output.py

```diff
@@ -25,6 +29,7 @@ def get_json_output(
             response.chunk.type == "content_block_delta"
             and (delta := response.chunk.delta)
             and delta.type == "input_json_delta"
+            and delta.partial_json
```

Comment:
this check is unnecessary since the following line returns `""` and if this check is removed but `delta.partial_json` is the empty string the return will be the same.

Reply:
done